server/src/subscriber/subscriber.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, Res } from "@nestjs/common";
+import { ConfigService } from "../config/config.service";
+import { Client } from "@sendgrid/client";
+import crypto from 'crypto';
+
+
+type HttpMethod = 'get'|'GET'|'post'|'POST'|'put'|'PUT'|'patch'|'PATCH'|'delete'|'DELETE';
```

Comment:
Note for posterity - It's not ideal to define this type here because it's copying a type from Sendgrid's package but they don't expect theirs and I couldn't find a better quick fix either.

Reply:
Yep, I would have imported if possible.